tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -7658,9 +7658,6 @@
     ]
    ]
   },
-  "stub": {
-   "css/blockify_inline_element.html": []
-  },
```

Comment:
Why is `./mach update-manifest` updating that test?

Reply:
It's fixing previously-incorrect updates.